templates/core/terraform/main.tf

```
@@ -107,3 +107,10 @@ module "acr" {
   core_vnet             = module.network.core
   shared_subnet         = module.network.shared
 }
+
+module "state-store" {
+  source = "./state-store"
+  name = "tre-${var.resource_name_prefix}"
+  location = var.location
+  resource_group_name = azurerm_resource_group.core.name
+}
```

```suggestion
module "state-store" {
  source                = "./state-store"
  resource_name_prefix  = var.resource_name_prefix
  environment           = var.environment
  tre_id                = local.tre_id
  location              = var.location
  resource_group_name   = azurerm_resource_group.core.name
}
```
Suggest changing to the same pattern as the rest of the modules; also solves the naming convention that @marrobi mentioned. To be used together with change to the statestore.tf